locallibs/install.py

```diff
@@ -87,6 +87,19 @@ def install_extras(framework_path, version="2.7", requirements_file=None,
                    install_wheel=False, upgrade_pip=False):
     """install all extra pkgs into Python framework path"""
     print()
+    python_guard_path = os.path.expanduser("~/Library/Python/%s") % version
```

Comment:
After further investigation, it appears Python (or pip adds _this_ path if it exists):

`~/Library/Python/<version>/lib/python/site-packages`

So I'd be happier if your "python_guard_path" was that.

Reply:
done